### PR TITLE
Collapsed diffs test

### PR DESCRIPTION
**Pull request by waldofe**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -1,18 +1,80 @@
...
+    # - ninja test
+
+# flatpak:master:
+    # image: registry.gitlab.com/alatiera/gnome-nightly-oci/gnome-master:latest
+    # stage: test
+    script:
+    - flatpak-builder --stop-at=nautilus app build-aux/flatpak/org.gnome.Nautilus.json
+    # Make sure to keep this in sync with the Flatpak manifest, all arguments
+    # are passed except the config-args because we build it ourselves
+    - flatpak build app meson --prefix=/app --libdir=/app/lib -Dprofile=development -Dtests=all _build
```
Comment (waldofe) [inline .gitlab-ci.yml]:
```suggestion
    - flatpak
```

```suggestion
    - flatpak build app meson --prefix=/app --libdir=/app/lib -Dprofile=development -Dtests=z
```

File: build-aux/flatpak/org.gnome.Nautilus.json [hunk 2]
```diff
@@ -84,9 +63,8 @@
             "builddir": true,
             "name": "nautilus",
```
Comment (oswaldoferreira) [inline build-aux/flatpak/org.gnome.Nautilus.json]:
```suggestion
            "name": unchanged line
```

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -1,18 +1,80 @@
-image: fedora:27
+# variables:
```
Comment (waldofe) [inline .gitlab-ci.yml]:
```suggestion
# variablevvvcvc
```

File: .gitlab-ci.yml [hunk 4]
```diff
@@ -1,18 +1,80 @@
...
+.test_template: &distro_test
   script:
-    - meson _build .
-    - ninja -C _build install
+    # - meson _build .
+    # - cd _build
+    # - ninja test
+
+# flatpak:master:
+    # image: registry.gitlab.com/alatiera/gnome-nightly-oci/gnome-master:latest
```
Comment (waldofe) [inline .gitlab-ci.yml]:
```suggestion
**aaa**
foo
bar
kux

editing second
```

```suggestion
foo
bar
editing first
editing first 2
```

out of the suggestion

File: build-aux/flatpak/org.gnome.Nautilus.json [hunk 5]
```diff
@@ -80,13 +59,14 @@
...
-            "name": "nautilus",
+
+            "name": "autilus",
             "config-opts": [
-                "-Denable-desktop=false",
-                "-Denable-selinux=false",
-                "--libdir=/app/lib"
+                "--libdir=/app/lib",
+                "-Dprofile=development"
             ],
```
Comment (waldofe) [inline build-aux/flatpak/org.gnome.Nautilus.json:92]:
```suggestion
            ],
            d
```

File: .gitlab-ci.yml [hunk 6]
```diff
@@ -1,18 +1,81 @@
...
-  - dnf install -y gcc meson gettext itstool redhat-rpm-config git
-                   gtk3-devel gnome-autoar-devel gnome-desktop3-devel
-                   gobject-introspection-devel gsettings-desktop-schemas-devel
-                   libselinux-devel libxml2-devel tracker-devel
-                   desktop-file-utils libgexiv2-devel
-  - pip install meson
-
-build-nautilus:
-  stage: build
+.test_template: &distro_test
```
Comment (oswaldoferreira) [inline .gitlab-ci.yml]:
```suggestion
.test_template: &distro_test
```

File: data/meson.build [hunk 7]
```diff
@@ -1,87 +1,154 @@
...
-                                       po_dir: po_dir,
-                                       type: 'desktop')
-endif
+#
+# This is primarily for the “build profiles” feature,
+# to be able to have icons even if we change the application ID.
+#
+# https://gitlab.gnome.org/GNOME/nautilus/merge_requests/144
+##########
+foreach icon_size: ['16x16', '22x22', '24x24', '32x32', '48x48', '512x512']
```
Comment (waldofe) [inline data/meson.build:7]:
```suggestion
foreach icon_size: ['16x16', '22x22', '24x24', '32x32', '48x48', '512x512']
```

File: data/org.gnome.nautilus.gschema.xml [hunk 8]
```diff
@@ -227,7 +226,7 @@
     <key type="as" name="captions">
       <default>[ 'none', 'none', 'none' ]</default>
       <summary>List of possible captions on icons</summary>
-      <description>A list of captions below an icon in the icon view and the desktop. The actual number of captions shown depends on the zoom level. Some possible values are: “size”, “type”, “date_modified”, “owner”, “group”, “permissions”, and “mime_type”.</description>
+      <description>A list of captions below an icon in the icon view. The actual number of captions shown depends on the zoom level. Some possible values are: “size”, “type”, “date_modified”, “owner”, “group”, “permissions”, and “mime_type”.</description>
```
Comment (waldofe) [inline data/org.gnome.nautilus.gschema.xml:229]:
```suggestion
      <description></description?
```

File: .gitlab-ci.yml [hunk 9]
```diff
@@ -1,18 +1,81 @@
...
-stages:
- - build
+# stages:
+# - test
+# - cross_distro
 
-before_script:
-  - dnf install -y gcc meson gettext itstool redhat-rpm-config git
-                   gtk3-devel gnome-autoar-devel gnome-desktop3-devel
-                   gobject-introspection-devel gsettings-desktop-schemas-devel
```
Comment (oswaldoferreira) [inline .gitlab-ci.yml]:
```suggestion
bar
```

File: .gitlab-ci.yml [hunk 10]
```diff
@@ -1,18 +1,81 @@
-image: fedora:27
+# ariables:
+#   GIT_SUBMODULE_STRATEGY: normal
 
-stages:
- - build
+# stages:
+# - test
```
Comment (oswaldoferreira) [inline .gitlab-ci.yml]:
```suggestion
# - test
```

File: .gitlab-ci.yml [hunk 11]
```diff
@@ -1,18 +1,81 @@
-image: fedora:27
+# ariables:
```
Comment (waldofe) [inline .gitlab-ci.yml]:
```suggestion
# ariables
```

File: .gitlab-ci.yml [hunk 12]
```diff
@@ -1,18 +1,81 @@
...
+#   GIT_SUBMODULE_STRATEGY: normal
 
-stages:
- - build
+# stages:
```
Comment (waldofe) [inline .gitlab-ci.yml]:
```suggestion
# stages:d
```

File: .gitlab-ci.yml [hunk 13]
```diff
@@ -1,18 +1,81 @@
-image: fedora:27
+# ariables
```
Comment (waldofe) [inline .gitlab-ci.yml]:
```suggestion
test change
```

File: .gitlab-ci.yml [hunk 14]
```diff
@@ -1,18 +1,84 @@
...
-build-nautilus:
-  stage: build
+test change
+#   GIT_SUBMODULE_STRATEGY: normal
+
+# stages:d
+# - test
+# - cross_distro
+
+.test_template: &distro_test
```
Comment (waldofe) [inline .gitlab-ci.yml:12]:
```suggestion
.test_template: &distro_test test test
```

File: .gitlab-ci.yml [hunk 15]
```diff
@@ -1,18 +1,85 @@
...
-                   gtk3-devel gnome-autoar-devel gnome-desktop3-devel
-                   gobject-introspection-devel gsettings-desktop-schemas-devel
-                   libselinux-devel libxml2-devel tracker-devel
-                   desktop-file-utils libgexiv2-devel
-  - pip install meson
 
-build-nautilus:
-  stage: build
+
+test change
```
Comment (waldofe) [inline .gitlab-ci.yml:5]: foo

File: .gitlab-ci.yml [hunk 16]
```diff
@@ -1,18 +1,85 @@
-image: fedora:27
```
Comment (waldofe) [inline .gitlab-ci.yml:1]: foo

**Pull request closed without merging**